--- nosefart-2.0-mls/Makefile.orig	2004-04-04 23:48:11.000000000 +0200
+++ nosefart-2.0-mls/Makefile	2004-04-18 23:40:49.692227312 +0200
@@ -2,10 +2,11 @@
 # Configuration
 
 CC = gcc
-CFLAGS = 
-LDFLAGS = -lm
+CFLAGS = $(OPTFLAGS)
+LDFLAGS =
+LDADD = -lm
 PREFIX = /usr/local
-WANT_DEBUG=TRUE
+WANT_DEBUG=FALSE
 
 # nothing below here should need to be changed
 
@@ -24,7 +25,7 @@
 ifeq "$(WANT_DEBUG)" "TRUE"
 	CFLAGS += -ggdb
 else
-	CFLAGS += -O2 -fomit-frame-pointer -ffast-math -funroll-loops
+	CFLAGS += -fomit-frame-pointer -ffast-math -funroll-loops
 	DEBUG_OBJECTS =
 endif
 
@@ -105,7 +106,7 @@
 # The real heavy lifting
 
 $(BUILDTOP)/$(NAME): $(OBJECTS)
-	$(CC) $(NSFINFO_CFLAGS) $(LDFLAGS) -o $@ $^
+	$(CC) $(NSFINFO_CFLAGS) $(LDFLAGS) -o $@ $^ $(LDADD)
 
 $(BUILDDIR)/%.o: $(SRCDIR)/%.c 
 	$(CC)  $(NSFINFO_CFLAGS) -o $@ -c $<
